oppia_android_test.bzl

```
@@ -0,0 +1,45 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+def oppia_android_test(name, srcs, test_manifest, custom_package, resource_files,
+                       test_class, src_library_name):
+  '''
+  This macro exists as a way to set up a test in Oppia Android to be run with Bazel.
+  This macro creates a library for an individual test that is fed as a dependency into an
+  android_local_test() rule which configures the test to be run with Roboletric and Bazel.
```

```suggestion
  Creates an Oppia test target for running the specified test as an Android local test with Kotlin support. Note that this creates an additional, internal library.
```

My suggestion needs wrapping, but suggestion to simplify the wording a tad. (Note that specifying Bazel in documentation is unnecessary in most cases--Bazel is implied since this code lives in Bazel files).

---

Okay!